docs/src/SUMMARY.md

```diff
@@ -38,7 +38,7 @@
     - [OpenLayers](using-with-openlayers.md)
   - [Data source specific](using-with-data.md)
     - [Using Hosted PostgreSQL](recipes.md)
-    - [Setting up a basemap and overlaying data]() <!-- markdownlint-disable-line MD042 -->
+    - [Setting up a basemap and overlaying data](recipe-basemap-postgis.md)
```

Comment:
Seems a wrong place to put this here?

Data format/tile source
- mbtiles
- pmtiles
- postgres

Data Source

- OSM
-  private data
- etc...

The 6.0 is about **Data source specific**.
And The 6.2.1 is talking about **data format/tile source**,  
And The 6.2.2 is talking about not only data source, it's talking  about **use case of martin**.

Maybe move this new content to somewhere else? Or we need to adjust whole doc structure

Reply:
@nyurik

Reply:
"structurally" we have...

* external data sources
  * vector (osm/overture/proprietary)
  * raster (satelite, areal)
* tile formats (mlt, mvt, png, ...)
* container formats (mbtiles, pmtiles)
* storage (files, postgres, s3)
* data transformation (only postgres at this point)

not sure how this helps though... classification is hard :)

Reply:
I have changed `Data source specific` to `Tile source specific`.
This is what I meant the section to be.